src/lib/develco.ts

```diff
@@ -7,6 +7,40 @@ import {logger} from './logger';
 const NS = 'zhc:develco';
 
 export const develcoModernExtend = {
+    addCustomClusterManuSpecificDevelcoGenBasic: () => deviceAddCustomCluster(
+        'genBasic',
+        {
+            ID: 0x0000,
+            attributes: {
+                zclVersion: {ID: 0, type: Zcl.DataType.UINT8},
```

Comment:
@Koenkk The merging of these is not 100% clear to do.

Can I just not repeat all the ZCL genBasic attributes here and only add the 3 develco ones (L30-L32) here and have it properly merged?

Reply:
It will get merged by default, so no need to re-specify everything.